.editorconfig

```diff
@@ -8,9 +8,14 @@ indent_style = space
 indent_size = 2
 indent_style = space
 
-[*.cs]
-# Require "this." keyword qualification in code
-dotnet_style_qualification_for_field = true:suggestion
-dotnet_style_qualification_for_property = true:suggestion
-dotnet_style_qualification_for_method = true:suggestion
-dotnet_style_qualification_for_event = true:suggestion
+[*]
+
+# Microsoft .NET properties
+csharp_new_line_before_members_in_object_initializers = false
```

Comment:
Code style rules in general are... subjective.
I'm going to revert some of these rules to keep it consistent to other AF repositories.
Hope you won't take this personally. :)

Reply:
Ofc. It's your project so go for your preferred style :)